.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   ignorePatterns: [
     "**/node_modules/**/*",
     "**/dist/**/*",
+    "**/static/**/*",
```

Comment:
is this required to fulfill this PR's title? Otherwise could be done in a separate PR where it doesn't muddy things?

Reply:
Sounds good.